src/linkding.js

```diff
@@ -65,4 +65,21 @@ export async function testConnection(configuration) {
     .then(response => response.status === 200 ? response.json() : Promise.reject(response))
     .then(body => !!body.results)
     .catch(() => false);
+}
+
+export async function lookForCurrentBookmark(url) {
```

Comment:
```suggestion
export async function findBookmarkByUrl(url) {
```

This basically tries to find a bookmark by URL, so the name could reflect that. Apart from that it looks like this function could reuse the `search` function internally by passing the query and limit parameter.

Regarding the return value it would make sense to return either the bookmark that was found, or null, instead of an array. That way the underlying search logic stays encapsulated here, rather than having the form deal with the fact that the underlying API can return a list of results.